staging/src/k8s.io/client-go/tools/cache/reflector_test.go

```diff
@@ -216,9 +221,10 @@ func BenchmarkReflectorResyncChanMany(b *testing.B) {
 func TestReflectorHandleWatchStoppedBefore(t *testing.T) {
 	s := NewStore(MetaNamespaceKeyFunc)
 	g := NewReflector(&testLW{}, &v1.Pod{}, s, 0)
-	stopCh := make(chan struct{})
-	// Simulate the watch channel being closed before the watchHandler is called
-	close(stopCh)
+	_, ctx := ktesting.NewTestContext(t)
+	ctx, cancel := context.WithCancelCause(ctx)
+	// Simulate the context being canceled before the watchHandler is called
+	cancel(errors.New("don't run"))
```

Comment:
always falling over this "don't run". What does it mean?

Reply:
It's a free-form explanation for why the context is canceled. In this case, the code using the context isn't even meant to start, therefore I picked "don't run". Any string is better than "context canceled"...